### PR TITLE
Coin base

### LINKED ISSUE #8: Coins
Pacman can eat coins moving on them. New coin is generated upon eating one.

### PR DESCRIPTION
**Pull request by dadam356**

### CODE REVIEW

File: TeamSix/pacman.py [hunk 1]
```diff
@@ -75,6 +85,52 @@ def handle_key_events(self):
...
+
+    def draw(self):
+        for coin_pos in self.positions:
+            pygame.draw.circle(screen, COLOR_WHITE, coin_pos, 5)
+            pygame.display.update()
+
+
 if __name__ == '__main__':
     game = Pacman()
+    coins = Coins(NUMBER_OF_COINS)
```
Comment (david-czuri) [inline TeamSix/pacman.py:150]: Having a coin class and removing some non-pacman logic from the class of Pacman was a great idea. However the removed parts are hanging now. I believe making a Game class and moving any initialization logic into that could benefit us in the long run.
It should contain the game loop (the while thing Pacmans play function) and hold an instance of Pacman and Coins and call their functions

File: TeamSix/pacman.py [hunk 2]
```diff
@@ -21,28 +30,29 @@ def __init__(self):
...
+        #self.fps = pygame.time.Clock()
 
     def play(self):
         while True:
             self.handle_key_events()
             self.change_direction()
             self.step()
-
-            self.game_window.fill(COLOR_BLACK)
+            #screen.fill(COLOR_BLACK)
```
Comment (david-czuri) [inline TeamSix/pacman.py]: Removing this line caused that we have to draw a black circle after pacman moves. This logic seems overcomplicated to me. I say that drawing a black screen then pacman and coins is an easier and more miantainable approach.

File: TeamSix/pacman.py [hunk 3]
```diff
@@ -75,6 +85,52 @@ def handle_key_events(self):
...
+            cnt += 1
+            if rnd_num == 0:
+                position_x += 30
+            elif rnd_num == 1:
+                position_y += 30
+            elif rnd_num == 2:
+                position_x -= 30
+            elif rnd_num == 3:
+                position_y -= 30
+            if position_y > 900 or position_y < 0 or position_x > 900 or position_x < 0:
```
Comment (david-czuri) [inline TeamSix/pacman.py]: I suppose the 900 represents the screen size here if so, we already have a constant declared for that.

**Review by DEW211 — CHANGES_REQUESTED** [review]
Some general things.
Consistent variable naming is kept all the way, but we should avoid abbreviations.
The coin generation function works well, but I believe it can be better, I am open for discussion here.

**Pull request merged**